README.md

```diff
@@ -264,15 +264,70 @@ Update coming...
+|`learn_weights(**kwargs)` | See [Learning in DDLite](#learnopts) below |
 |`learn_weights_validated(**kwargs)` | Wrapper for `learn_weights()`|
 |`set_use_lfs(use=True)` | Setter for using LF weights in prediction |
 |`get_log_odds(subset=None)` | Get log odds values for all candidates (`subset=None`), an index subset (`subset`), test set (`subset='test'`), or validation set (`subset='validation'`) |
 |`get_predicted_probability(subset=None)` | Get predicted probabilities for candidates |
 |`get_predicted(subset=None)` | Get predicted responses for candidates |
+|`get_log_odds_lf(subset=None)` | Get log odds values for all candidates (`subset=None`), an index subset (`subset`), test set (`subset='test'`), or validation set (`subset='validation'`) using only LFs|
+|`get_predicted_probability_lf(subset=None)` | Get predicted probabilities for candidates using only LFs |
+|`get_predicted_lf(subset=None)` | Get predicted responses for candidates using only LFs|
+
 |`get_classification_accuracy(subset=None)` | Get classification accuracy over `subset` against ground truth |
 |`plot_calibration()` | Show DeepDive calibration plots |
 |`open_mindtagger(num_sample=None, abstain=False, **kwargs)` | Open MindTagger portal for labeling; sample is either the last sample (`num_sample=None`) or a random sample of size `num_sample`; if `abstain=True` show only canidates which haven't been labeled by any LF |
 |`add_mindtagger_tags()` | Import tags from current MindTagger session |
 |`add_to_log(log_id=None, subset='test', show=True)` | Add current learning result to log; precision and recall computed from ground truth in `subset` |
 |`show_log(idx=None)` | Show all learning results (`idx=None`) or learning result `idx` | 
+
+## Learning in DDLite<a name="learnopts"></a>
+DDLite provides a number of options for learning predictive models. The simplest option is to just `DDLiteModel.learn_weights()`, but the results probably won't be so hot using only default values. There are two important decisions to make before setting other parameters.
+
+* **Pipeline or joint learning?** In the pipeline learning approach, the LF accuracies are learned separately from the feature weights. These are the "LF stage" and "feature stage", respectively. In the joint learning approach, the LF accuracies and feature weights are learned at the same time. We suggest using pipeline (`pipeline=True`) since it's more flexible than the joint approach (`pipeline=False`); pipeline is also the default.
```

Comment:
Instead of "learned separately", something indicating sequential, more like "learned first, then..."

O/w can edit this more later